src/config.coffee

```diff
@@ -7,38 +7,38 @@ checkInt = (s) ->
 		return
 	return i
 
-checkValidKey = (s) ->
+checkString = (s) ->
 	# Make sure `s` exists and is not an empty string, or 'null'.
-	if !s? or s == 'null' or s == ''
+	if !s? or s == 'null' or or s == 'undefined' or s == ''
```

Comment:
I really don't know, but I wanted to proactively protect from possible jq weirdness.
I'll add the comment and merge, cheers